wordpress-mobile/WordPress-iOS
# Site Settings: Add the ability to edit the Site Icon

Fixes #6905 

To test:
1. Select one of your sites
2. Tap on the site icon and then:
   2.1. Select an image from your Camera Roll, Crop it and Use it.
   2.2. Select an image from your Media Library, Cropt it and Use it.

For each update you can check that everything also looks good on Calypso.

Please note that for now this will only work on WP.com sites due to an issue reported here: https://github.com/wordpress-mobile/WordPress-iOS/issues/6802 
I will try to address that one separately.


Needs review: @frosty
